spec/services/unsubscribe_all_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe UnsubscribeAllService do
+  describe ".call" do
+    let(:subscriber) { create(:subscriber, address: "foo@bar.com") }
```

Comment:
Minor, this email address seems redundant.

Reply:
It is! Was a copy over from an existing test which has now been removed.